developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <d:resourcetype />            | Specifies the nature of the resource.            | ``<d:collection />`` for a folder                                        |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <d:getcontentlength />        | The size if it is a file.                        | ``3030237``                                                              |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <d:getcontentlanguage />      | The language of the content.                     | ``en``                                                                   |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <d:displayname />             | A name suitable for presentation.                | ``File name``                                                            |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <d:lockdiscovery />           | Describes the active locks on a resource.        | | [ {                                                                    |
+|                               |                                                  | |      locktype: ``write``                                               |
+|                               |                                                  | |      lockscope: ``exclusive``                                          |
+|                               |                                                  | |      depth: ``0``                                                      |
+|                               |                                                  | |      owner: ``Jane Smith``                                             |
+|                               |                                                  | |      timeout: ``Infinite``                                             |
+|                               |                                                  | |      locktoken: { href: ``urn:uuid:f81de2ad-7f3d-a1b2-4f3c-`` }        |
+|                               |                                                  | |      lockroot: { href: ``http://www.example.com/container/`` }         |
+|                               |                                                  | | } ]                                                                    |
```

Comment:
This example makes it look like the content would be json, but https://github.com/sabre-io/dav/blob/master/lib/DAV/Xml/Property/LockDiscovery.php writes xml. Could you make the example also use xml?

Reply:
Update as it is actually a dummy endpoint.